src/workflow_review.md

```diff
@@ -122,7 +134,18 @@ For screenshots, see [here](https://inbo.github.io/git-course/workflow_rstudio.h
 
 <div class="alert alert-danger">
 When merging to the master branch or another important branch in the repo,
-always do this on GitHub by using a pull request.
+always do this on GitHub by using a pull request. It is recommended to protect
+these branches to enforce this procedure.
 </div>
 
 Once you are satisfied with a version, you can [create a release](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository).
+
+# Branch management
+
+After deleting branches on the remote, you may want to get rid of them as well
+in the selection list in the git pane in RStudio:
+
+- hit `More` and choose `Shell...`
```

Comment:
I'm planning on adding a `clean_up()` function in `checklist`. inbo/checklist#46

Reply:
Nice!  I guess this was the only annoying task that was still remaining.  :-D

But maybe I better wait to change it till it is up and running?

Reply:
Yes. Keep the current text as is. Merge when ready. Don't wait on inbo/checklist#46

Reply:
I'm afraid I'm not able to merge myself, as I don't have writing permits to this repo. :-(